Views/Layout.cshtml

```diff
@@ -1,17 +1,38 @@
 <!DOCTYPE html>
+@using Microsoft.Extensions.Options
+@using OrchardCore.ResourceManagement
 
 @inject ICssClassHolder CssClassHolder
+@inject IOptions<ResourceManagementOptions> ResourceManagementOptions
 
 <html lang="@Orchard.CultureName()">
 <head>
+    @{
+        // Remove all external Bootstrap 5 resources. (This way we can update Bootstrap independent of OC.)
+        foreach (var manifest in ResourceManagementOptions.Value.ResourceManifests)
+        {
+            if (manifest.GetResources("$" + nameof(FeatureIds.Area)).ContainsKey(FeatureIds.Area)) { continue; }
+
+            foreach (var resourceType in new[] { "stylesheet", "script" })
+            {
+                if (manifest.GetResources(resourceType).TryGetValue("bootstrap", out var resource))
```

Comment:
Isn't it possible to just define a new version for the `bootstrap` resource? Would that mess up the Admin UI?

Reply:
> Isn't it possible to just define a new version for the bootstrap resource?

We already define a new resource [here](https://github.com/Lombiq/Orchard-Base-Theme/pull/3/files/09074b82b2647a700b6cef3169e3d5b71f911e46#diff-1108edf78b508be566482baa4206490628eabec71eecfb75309df1d97d5f8973R28-R30). If you define a new resource it gets added to a separate resource dictionary by module. (This is the "new" behavior introduced in 1.0, probably to avoid people trying to edit already added resources during configuration, which would be fragile and load order dependent.) What this logic does is removes every other bootstrap resource that wasn't added in this module.

> Would that mess up the Admin UI?

It won't affect the Admin UI because the base theme is not an admin theme.

Reply:
OK